src/platform/user/authentication/components/LoginButton.jsx

```diff
@@ -10,9 +10,8 @@ function loginHandler(loginType) {
   });
 }
 
-export default ({ csp, isDisabled }) => (
+export default ({ csp }) => (
```

Comment:
`isDisabled` was the direct value of `globalDowntime`. Stripping it out along side it